lib/fog/ovirt/requests/compute/v4/add_volume.rb

```diff
@@ -0,0 +1,54 @@
+module Fog
+  module Compute
+    class Ovirt
+      class V4
+        class Real
+          # rubocop:disable Style/NumericLiterals
+          DISK_SIZE_TO_GB = 1073741824
```

Comment:
Writing this as `DISK_SIZE_TO_GB = 1024 * 1024 * 1024` would be a bit more readable